grouper/constants.py

```diff
@@ -28,6 +31,9 @@
     (PERMISSION_CREATE, "Ability to create permissions within Grouper."),
     (PERMISSION_GRANT, "Ability to grant a permission to a group."),
     (PERMISSION_AUDITOR, "Ability to own or manage groups with audited permissions."),
+    (PERMISSION_ADMIN, "Ability to manipulate any permission."),
+    (USER_ADMIN, "Ability to to disable/enable any User account and modify its attributes."),
+    (GROUP_ADMIN, "Ability to approve/deny/revoke membership to any group."),
```

Comment:
I've kept them in `SYSTEM_PERMISSIONS` but simply enumerate the permissions to automatically grant during `sync_db` in 0fd2a16.